python/ray/serve/scripts.py

```diff
@@ -402,22 +438,84 @@ def run(
     type=str,
     help=RAY_DASHBOARD_ADDRESS_HELP_STR,
 )
-def config(address: str):
-
-    app_info = ServeSubmissionClient(address).get_info()
-    if app_info is not None:
-        print(yaml.safe_dump(app_info, sort_keys=False))
+@click.option(
+    "--all-apps",
```

Comment:
Could you rename this option? `-a` here clashes with the `-a` for the `--address` option.

Reply:
I've renamed it to `--multi-app`. This also matches the flag on `serve build` for consistency.